### ISSUE TITLE
Purchase does not work | transaction is always null

### ISSUE THREAD

**Issue by SENPAY98K** [chr233/ASFEnhance#119]
**Describe the bug / Bug 描述**
Transaction is always null..

```txt
ASFenhance 遇到错误, 日志如下
==========================================
 - 原始消息: TEST
 - Access: Owner
 - ASF 版本: 5.4.7.3
 - 插件版本: 1.8.4.0
==========================================
{
  "EULA": true,
  "Statistic": true,
  "DevFeature": false
}
==========================================
```

**Comment by Watashi-Senpai** [comment]
+1

**Comment by chr233** [comment]
steam now has change its checkout domain to `checkout.steampowered.com`, and ASF don't store cookies under this domain

**Comment by SENPAY98K** [comment]
> steam now has change its checkout domain to `checkout.steampowered.com`, and ASF don't store cookies under this domain

So there is no work around it ?

Also i had create it previously with node, but after this change, i could not get it any success for the `initialtransaction` request to get the transaction id ! do you have any idea as well !

**Comment by chr233** [comment]
there is a way to auto login to `checkout.steampowered.com`, but it takes too much work, currently I cant provide such effort on this project, I have works to do.

**Comment by SENPAY98K** [comment]
> 

Yes, i am aware of that,tried it and could not get to succeed! always getting `null results with success:2`!

As you mentioned about ASF storing cookies for domain `store` , i assume that my fails related to cookies only which im getting correctly! else there should be a specific cookie!

Thanks for the responses :)

**Comment by chr233** [comment]
new domain `checkout.steampowered.com` has its CORS rule, and sessionid doesn't same as `store.steampowered.com`, same as the `help.steampowered.com` , these are three different domain

**Comment by SENPAY98K** [comment]
> new domain `checkout.steampowered.com` has its CORS rule, and sessionid doesn't same as `store.steampowered.com`, same as the `help.steampowered.com` , these are three different domain

Oh good to know! as far i know, i checked the browser and i did not see anything different ! else i'm looking at wrong place

**Comment by chr233** [comment]
![image](https://github.com/chr233/ASFEnhance/assets/13798576/7335bcde-1bd4-4271-a466-493d1d893602)
its seems that the sessionId is common to use

**Comment by chr233** [comment]
fixed in 1.8.5.0

**Comment by SENPAY98K** [comment]
Thanks for the fix! it works fine at the moment.
but there a message related to asf web requests, is it safe!
![image](https://github.com/chr233/ASFEnhance/assets/77079047/f67cef26-d14e-4d44-86be-668caa681ed0)

**Comment by chr233** [comment]
https://github.com/JustArchiNET/ArchiSteamFarm/pull/2934

pull requested to ASF,

**Comment by SENPAY98K** [comment]
> [JustArchiNET/ArchiSteamFarm#2934](https://github.com/JustArchiNET/ArchiSteamFarm/pull/2934)
> 
> pull requested to ASF,

Thanks for making things clear, I would not do it since ASF does not support issues when using plugins.

**Comment by chr233** [comment]
> Thanks for the fix! it works fine at the moment.
> but there a message related to asf web requests, is it safe!
> ![image](https://github.com/chr233/ASFEnhance/assets/77079047/f67cef26-d14e-4d44-86be-668caa681ed0)
> 

fix in 1.8.6.1 under ASF 5.4.7.3，and after ASF 5.4.8.0, this is fixed in original ASF

**Comment by SENPAY98K** [comment]
> > Thanks for the fix! it works fine at the moment.
> > but there a message related to asf web requests, is it safe!
> > ![image](https://user-images.githubusercontent.com/77079047/249405269-f67cef26-d14e-4d44-86be-668caa681ed0.png)
> 
> fix in 1.8.6.1 under ASF 5.4.7.3，and after ASF 5.4.8.0, this is fixed in original ASF

Noice, well done 😄
